framework/caching-cmm.md

```diff
@@ -147,10 +148,12 @@ If a cache entry is found, the caching CMM MUST return the encryption materials
 If a cache entry is not found or the cache entry is expired, the caching CMM MUST then attempt to obtain the encryption materials
 by making a call to the underlying CMM's [Get Encryption Materials](cmm-interface.md#get-encryption-materials).
 
-If the [algorithm suite](algorithm-suites.md) requested does not contain an [Identity KDF](algorithm-suites.md#identity-kdf),
+If the [algorithm suite](algorithm-suites.md) is present in the encryption materials request,
+and it does not contain an [Identity KDF](algorithm-suites.md#identity-kdf),
 the caching CMM MUST add the encryption materials obtained from the underlying CMM into the underlying CMC.
```

Comment:
This is missing the "If no algorithm suite is present" case, in which we still want to add the materials to the underlying CMM (using the no-alg identifier explained below).

Reply:
Good catch. Does this look good?

```suggestion
If the [algorithm suite](algorithm-suites.md) is present in the encryption materials request
and does not contain an [Identity KDF](algorithm-suites.md#identity-kdf),
or if the algorithm suite is not present,
then the caching CMM MUST add the encryption materials obtained from the underlying CMM into the underlying CMC.
```

Reply:
LGTM. thanks!